scripts/server

```diff
@@ -16,6 +16,8 @@ FLASK_ENV=development
 # Start Worker and Server. Whatchdog detects any change to the .py files and restarts celery
 pipenv run watchmedo auto-restart -d . -p '*.py' -- celery worker -E -B -A inspirehep.celery -l INFO & pid_celery=$!
 
+pipenv run gunicorn --reload -b 0.0.0.0:5000  --access-logfile "-" --error-logfile - inspirehep.wsgi:application & pid_server=$!
```

Comment:
is `server` script even relevant? I thought we were supposed to use `docker` for dev?

Reply:
I realized that this is used as a command for `worker` service. And now it runs `gunicorn` in worker service too. This might be the issue why `e2e` needed more timeout!

Reply:
Good one, let me check this

Reply:
it worked, I refactored the scripts